### ISSUE TITLE
Eating items

### ISSUE THREAD

**Issue by VoidLock92** [williambl/boats-and-beeps#2]
Not sure if this is normal but this slot just eats whatever item I put in it regardless.
![image](https://user-images.githubusercontent.com/21327041/127708848-b8325ca4-5b6d-4570-959c-c93d9bbbe0f2.png)

**Comment by LostSoul3471** [comment]
Pretty sure that slot is only for additional boats to add more parts to it, in other words to make a boat longer.  I figured that out the hard way on the initial version.  Might want to code in something to validate what gets dropped there and better docs, eventually. :)

**Comment by VoidLock92** [comment]
Yeah, you might be right. Just thought I would open a ticket because maybe it is not supposed to accept everything and only what will have an effect? I'm not sure :P Right now its a black hole if you give it the wrong items haha
